.gitignore

```diff
@@ -5,6 +5,8 @@ yarn.lock
 **/*.log
 test/repo-tests*
 
+.vscode
+.eslintrc
```

Comment:
Is there any reason we need to ignore the `. eslintrc` file if its not being used? Usually aegir takes care of all our configs, linting included.

Reply:
editors need to find a eslint config to provide linter messages and fix problems, because the config is hidden inside the `aegir > eslint-config-aegir > .eslintrc` the editor cant find it.
because of this i need to manually add this file in every repo to have a normal ctrl-s workflow.

Reply:
Editors will usually allow pointing their linting tools to custom locations, and with many contributors/editors being used I don't think it would be practical to add a custom entry for each to the `.gitignore` file.

I don't know what our current guidelines in regards to `.gitignore` are, maybe @diasdavid or @VictorBjelkholm can provide more context.

Reply:
This works for all editors because this is all based on how eslint works, nothing fancy here 😄